src/org/omegat/core/spellchecker/ISpellCheckerProvider.java

```diff
@@ -50,6 +51,12 @@ public interface ISpellCheckerProvider {
      */
     void learnWord(String word);
 
+    /**
+     * Initialize checker.
+     * @throws SpellCheckerException if the spell checker can't be initialized for this language.
+     */
+    void init(String language) throws SpellCheckerException;
```

Comment:
```suggestion
    boolean init(String language) throws SpellCheckerException;
```

It is better to return boolean other than throw Exception when spellchecker does not support language.